domains/authorization/README.md

```diff
@@ -64,6 +64,5 @@ You must manually remember to generate the complete SQL migration script after a
 This is the commands for generating the migration SQL script for the API project and Worker project:
 
 ```shell
-dotnet ef migrations script --idempotent --project Query.API/API --output migrations/API.sql
-cp migrations/API.sql migrations/Worker.sql
+dotnet ef migrations script --idempotent --project Authorization.API/API --output migrations/API.sql
```

Comment:
Can you update the Readme to remove mentions of Worker.sql, for the whole Readme file